ci/release/README.md

```diff
@@ -0,0 +1,140 @@
+
+- Sonatype OSSRH Artifacts:
+  - https://s01.oss.sonatype.org/content/repositories/{snapshots,staging,releases}/substrait/substrait-java/core/1.0.0/
+
+## Branches Configuration
+
+- In case of new features or breaking changes create a branch with any format name.
+  Example:
+  ```shell
+  # Current version on production: 2.5.7
+  $ git checkout -b new_feature_1234 main
+  ```
+  After your PR is merged to main branch, you could see the new libraries available at Sonatype staging environment.
+
+
+- In case of maintenances please create a branch name with this pattern `+([0-9])?(.{+([0-9]),x}).x`.
+  Example:
+  ```shell
+  # Current version on production: 2.5.7, version planned to use for maintenance: 2.1.7
+  $ git checkout -b 2.1.x v2.1.7 # for fix
+  $ git checkout -b 2.x v2.1.7 # for maintenance
+  ```
+  After your PR is merged to main branch, you could see the new libraries available at Sonatype staging environment.
+
+
+- In case you need to experiment with breaking change functionalities for new feature, take a look at pre-release branch.
+  Example:
+  ```shell
+  # Current version on production: 2.5.7, version planned to use for maintenance: 2.1.7
+  $ git checkout -b new_feature_lots_changes beta
+  ```
+  After your PR is merged to main branch, you could see the new libraries available at Sonatype snapshots environment.
+
+
+Branches configuration:
+
+```json
+  "branches": [
+    // maintenances (also generate release)
+    { "name": "+([0-9])?(.{+([0-9]),x}).x" },
+    // release
+    { "name": "main" },
+    { "name": "next" },
+    { "name": "next-major" },
+    // pre-release
+    { "name": "beta", "prerelease": true },
+    { "name": "alpha", "prerelease": true }
+  ],
+```
+
+## Q&A
+
+#### 1. What will be happened if I merge something on `main` branch and finalize with errors?
+
+CICD has a token configure with granted access to create issues as needed:
+
+```shell
+[10:30:55 PM] [semantic-release] [@semantic-release/github] › ℹ  Created issue #3: https://github.com/substrait/sustrait-java/issues/3.
+```
+
+![release_failing](img/automate_release_is_failing.png)
+
+#### 2. Could be possible to implement a process to deliver a library with custom version (i.e.: 3.2.9.RC1, 5.0.0.M1)?
+
+We are using Sonatype OSSRH and there are repositories for `snapshots` / `staging` / `release`:
+- If your library finalize with `1.4.3-XXXXX-SNAPSHOT` -> it is going to be deployed to Snapshots repository
+- If your library finalize with `1.4.3-XXXXX-SNAPSHOT.YYYY` -> it is going to be deployed to Staging/Releases repository
+- If your library finalize with `1.4.3-XXXXX` -> it is going to be deployed to Staging/Releases repository
+- If your library finalize with `1.4.3` -> it is going to be deployed to Staging/Releases repository
+
+#### 2. How many days do I have to approve the artifacts on the Staging repository?
+
+Please review [OSSRH-24751](https://issues.sonatype.org/browse/OSSRH-24751)
```

Comment:
What happens if a release is not approved in the time limit?

Reply:
There is not an option to recover that version. A initial proposal is to run a Job that after 5 days on staging environment automatically deploy that to release maven central, more detal at https://github.com/substrait-io/substrait-java/pull/102#issuecomment-1311059317

Reply:
Consider: [OSSRH-86341](https://issues.sonatype.org/browse/OSSRH-86341)

- Open staging repositories are dropped after 3 days of inactivity.
- Closed staging repositories are dropped after 21 days of inactivity.
- Released staging repositories are dropped within an hour or two of release.